resources/views/createTableMigration.php

```diff
@@ -26,15 +27,15 @@
 ?>
 
 use Yiisoft\Yii\Db\Migration\MigrationBuilder;
-use Yiisoft\Yii\Db\Migration\RevertibleMigrationInterface;
+<?= $transactional === false ? 'use Yiisoft\Yii\Db\Migration\RevertibleMigrationInterface' : 'use Yiisoft\Yii\Db\Migration\TransactionalMigrationInterface' ?>;
 
 /**
  * Handles the creation of table `<?= $table ?>`.
 <?= $this->render(__DIR__ . '/_foreignTables.php', [
     'foreignKeys' => $foreignKeys,
 ]) ?>
  */
-final class <?= $className ?> implements RevertibleMigrationInterface
+final class <?= $className ?> implements <?= $transactional === false ? "RevertibleMigrationInterface\n" : "TransactionalMigrationInterface\n" ?>
```

Comment:
```suggestion
final class <?= $className ?> implements <?= $transactional ? 'TransactionalMigrationInterface' : 'RevertibleMigrationInterface' ?>
```

Reply:
If you remove "\n" the newline, the tests will fail.